src/js/player.js

```diff
@@ -311,11 +354,72 @@ class Player extends Component {
     this.tech = false;
   }
 
+  addTechControlsListeners() {
+    // Some browsers (Chrome & IE) don't trigger a click on a flash swf, but do
+    // trigger mousedown/up.
+    // http://stackoverflow.com/questions/1444562/javascript-onclick-event-over-flash-object
+    // Any touch events are set to block the mousedown event from happening
+    this.on(this.tech, 'mousedown', this.handleTechClick);
+
+    // If the controls were hidden we don't want that to change without a tap event
+    // so we'll check if the controls were already showing before reporting user
+    // activity
+    this.on(this.tech, 'touchstart', this.handleTechTouchStart);
+    this.on(this.tech, 'touchmove', this.handleTechTouchMove);
+    this.on(this.tech, 'touchend', this.handleTechTouchEnd);
+
+    // Turn on component tap events
+    this.tech.emitTapEvents();
```

Comment:
This should really be handled by the tech. Possibly turned on for all techs, unless there's a reason not to with players like youtube.